src/server/protocol.ts

```diff
@@ -3007,6 +3007,12 @@ namespace ts.server.protocol {
          * For those entries, The `insertText` and `replacementSpan` properties will be set to change from `.x` property access to `["x"]`.
          */
         readonly includeCompletionsWithInsertText?: boolean;
+        /**
+         * Unless this option is disabled, or `includeCompletionsWithInsertText` is not enabled,

```

Comment:
```suggestion
         * Unless this option is explicitly false, or `includeCompletionsWithInsertText` is not enabled,
```

Agree, here's a button to do this ^